Formula/hh-suite.rb

```diff
@@ -18,11 +18,10 @@ class HhSuite < Formula
 
   uses_from_macos "perl"
 
-  fails_with :clang # needs openmp
-
   on_macos do
     depends_on "gcc" # needs openmp
   end
+  fails_with :clang # needs openmp
```

Comment:
The `on_macos` should be the last block before `def install`. There's no automated style check for this yet, but I believe it's planned.